### ISSUE TITLE
woocommerce main page (/shop) problems

### ISSUE THREAD

**Issue by androos** [phpbits/widget-options#34]
On the woocommerce main page /shop the visibility doesn't work right. some widget are visible but set to invisible other are set to invisible but visible. On other paths it works.

**Comment by phpbits** [comment]
@androos  Thank you very much for choosing Widget Options! Regarding your concern, have you tried using `is_shop()` on display logic option. Let me know how it goes. Thanks!

Cheers,
Jeffrey

**Comment by androos** [comment]
Is there also an "is_not..." option ?

**Comment by phpbits** [comment]
@androos you can use `!is_shop()`. Let me know how it goes. Thanks!

**Comment by androos** [comment]
I don't know, bt some thing is wrong here. 
I have a product search widget and a standard search widget.
Standard search should be invisible on /shop
Product search should be visible on /shop
Standard search was setup with Hide/Show = "Show" and in Post types "Shop" selected
Product search was setup with Hide/Show = "Hide" and in Post types "Shop" selected
Result: Standard search widget is shown on every page

Then I have added is_shop() respectively !is_shop()
Result: Product search is shown on every page.

**Comment by androos** [comment]
Correction, after using is_shop() respectively !is_shop(), product search is visible on shop and not visible on other pages, standard search never visible.

**Comment by phpbits** [comment]
@androos on product search add `is_shop()` then on the standard search widget add `!is_shop()`. You may need to clear your cache also. Let me know how it goes. Thanks!

**Comment by androos** [comment]
Yes !is_shop() is for the standard search, because it shouldn't be shown, and is_shop() is set on the product search widget, which should be shown. Cache is cleared, forced reload used. Behavior is as described, have tested it back an forth.

**Comment by androos** [comment]
I have similar problems with all other Widget Visibility plugins. Everything works as expected, except when the main shop page comes in to play.

**Comment by stebentje** [comment]
I have the same issue.

I managed that my widget is only shown in my shop area. I have following selected:

Post types:
‘Shop’
‘Cart’
‘Whishlist’

Custom Post Types:
‘Products’

This is working perfectly, only on my main shopping page the widget is not showing. Does anybody knows this issue?

PS: is_shop() on display logic option is not working
